shared/lib_cec6par.cpp

```diff
@@ -480,6 +480,11 @@ bool mcsp_celltemp_t::operator() ( pvinput_t &input, pvmodule_t &module, double
 
 	// convert to kelvin
 	double TA = input.Tdry+273.15;
+    if (isnan(input.Patm)) {
```

Comment:
Please use std::isnan instead of isnan, see https://github.com/NREL/SAM/issues/1223